cmd/norduser/main.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
+	<-make(chan struct{})
```

Comment:
Is this the same as `time.Sleep(infinite)` ? What is the purpose of such non-functional main?

Reply:
Just for testing purposes, will be removed once main is implemented.

Reply:
if it's WIP then why PR ?

Reply:
Make PR fit in <1k changes.